### PR TITLE
Url lookup

### PR DESCRIPTION
**Pull request by bcoca**
now you can loop over what a url returns, with the same usability and security concerns of piping curl/wget into your shells!

This does nothing fancy, can easily be expanded to handle specific needs of different URLS (ftp authentication? https? etc).

### CODE REVIEW

File: docsite/rst/playbooks_lookups.rst [hunk 1]
```diff
@@ -128,6 +128,10 @@ Here are some examples::
 
          - debug: msg="{{ lookup('etcd', 'foo') }} is a value from a locally running etcd"
 
+         - debug: msg="{{item}}
```
Comment (resmo) [inline docsite/rst/playbooks_lookups.rst]: there is a " missing

**Comment by resmo** [comment]
Would this also work with other protocols? e.g. FTP?

**Comment by bcoca** [comment]
yes, tested with   'ftp://ftp.us.debian.org/debian/README', for example

**Comment by bcoca** [comment]
these were my tests:

```
- hosts: localhost
  gather_facts: false
  connection: local
  tasks:
    - debug: msg="{{item}}"
      with_url:
        - 'http://github.com/otherguy.keys'
        - 'ftp://ftp.us.debian.org/debian/README'
        - 'file:///tmp/test1'
```

**Comment by resmo** [comment]
nice!

**Comment by cemo** [comment]
nice

**Pull request merged**